code/modules/core_implant/cruciform/rituals/priest.dm

```diff
@@ -806,7 +806,7 @@
 			new /obj/item/organ/external/robotic/church/l_leg(altar.loc)
 			return TRUE
 		if (response == "Shortsword")
```

Comment:
```suggestion
		if (response == "Longsword")
```

Reply:
And don't forget the Shortsword in 'What limb do you require', which I can't suggest change for.